flake.nix

```diff
@@ -9,15 +9,22 @@
     lib = {
       hm = import ./modules/lib { lib = nixpkgs.lib; };
       homeManagerConfiguration = { configuration, system, homeDirectory
-        , username, extraSpecialArgs ? { }
```

Comment:
I find the indentation of these args kind of confusing. What's the idea behind the grouping?

Reply:
I kind of agree, that is how `nixfmt` want to format things (which is used by this project). It will wrap arguments and group them as many as can fit. The only way to affect it is to add extra white space in between and group them manually in that manner.

Reply:
Other than that arguments are with the non-optional first